### PR TITLE
Create starting point for AI Profile

### PR DESCRIPTION
**Pull request by kestewart**
Add in the AI.md place holder for the profile.

- Signed-off-by: Kate Stewart <kate.stewart@att.net>

### CODE REVIEW

**Comment by rgopikrishnan91** [comment]
This satisfies what we want for the profile

**Review by goneall — APPROVED** [review]
LGTM - but I would wait for @zvr to review before merging

**Review by zvr — APPROVED** [review]

**Pull request merged**
